### PR TITLE
Elena Volosevich: homework 1

### PR DESCRIPTION
**Pull request by LenX21**
chromedriver.exe  was added 
sample_script was updated with new executable_path

### CODE REVIEW

**Review by svetlannnka — COMMENTED** [review]
Great!

**Comment by LenX21** [comment]
Home work#2 Elena Volosevich:
home_work_2.py contains:  Task 2 : Practice with locators + Task 3: Create a test case for Help search
homwork2 folder contains: Test Case using BDD

File: home_work_2.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+Create your Amazon account button:
+By.ID, $x("//a[@id='createAccountSubmit']")
+
+
+*Conditions of use link:
+$x("//a[@class='a-link-normal' and contains(text(), 'Conditions of Use')]")
+
+*Privacy Notice link:
+$x("//a[contains(@href, 'ap_desktop_footer_privacy_notice')]")
```
Comment (svetlannnka) [inline home_work_2.py]: 👍

File: home_work_2.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+# 1. Open Amazon Help page https://www.amazon.com/gp/help/customer/display.html
+driver.get("https://www.amazon.com/gp/help/customer/display.html")
+
+# 2. Use “Find more solutions” field and search for Cancel order:
+search_help_library_text_field = driver.find_element(*SEARCH_HELP_LIBRARY_TEXT_FIELD)
+search_help_library_text_field.send_keys(search_value)
+
+# 3. Click Go - Replaced with Enter (Didn't find 'Go button')
+search_help_library_text_field.send_keys(Keys.ENTER)
```
Comment (svetlannnka) [inline home_work_2.py]: Perfect!

**Review by svetlannnka — COMMENTED** [review]
This is awesome! I don't have any comments, great 👏

**Pull request closed without merging**